provisioning/device/src/polling_state_machine.ts

```diff
@@ -25,41 +26,43 @@ export class  PollingStateMachine extends EventEmitter {
     this._transport = transport;
 
     this._fsm = new machina.Fsm({
-      namespace: 'provisioning-transport',
+      namespace: 'provisioning-client-polling',
       initialState: 'disconnected',
       states: {
         disconnected: {
           _onEnter: (err, result, response, callback) => {
-            this._currentOperationCallback = null;
             if (callback) {
               callback(err, result, response);
             }
           },
-          register: (request, callback) => {
-            this._fsm.transition('sendingRegistrationRequest', request, callback);
-          },
-          cancel: (err, result, response, callback) => {
-            /* Codes_SRS_NODE_PROVISIONING_TRANSPORT_STATE_MACHINE_18_025: [ If `cancel` is called while disconnected, it shall immediately call its `callback`. ] */
-            // nothing to do.
-            callback();
-          }
+          register: (request, callback) => this._fsm.transition('sendingRegistrationRequest', request, callback),
+          /* Codes_SRS_NODE_PROVISIONING_TRANSPORT_STATE_MACHINE_18_025: [ If `cancel` is called while disconnected, it shall immediately call its `callback`. ] */
+          cancel: (cancelledOpErr, callback) => callback(),
```

Comment:
>callback() [](start = 48, length = 10)

feels like we'd be swallowing the cancelledOpErr argument here - are we guaranteed that it's null here? should we throw if it isn't?